CVLByExamples/README.md

```diff
@@ -0,0 +1,15 @@
+
+
+These examples can serve as an introduction to CVL ( Certora Verification Language ).
+It also contains examples that are based on real-life code and the spec that have caught critical bugs.
+
+See https://docs.certora.com for a complete guide. 
+
+
+To run the examples, use the provided configuration 
+```
+cd ConstantProductPoolExample 
+certoraRun ConstantProductPoolExample/certora/conf/run.conf 
+```
+
+Alternatively the [Certora IDE VSCode Extension](https://marketplace.visualstudio.com/items?itemName=Certora.vscode-certora-prover):
```

Comment:
seems to be missing text

Reply:
waiting to see if vscode support this, currently not. so removing suggestion to install it